docs/user/metal/README.md

```diff
@@ -0,0 +1,22 @@
+# Support for Bare Metal Environments
+
+OpenShift has support for deployments on baremetal with either [User
```

Comment:
Per the suggested style guidelines, ttps://github.com/metal3-io/metal3-docs/pull/47, I think this should be `on bare metal hosts`

Reply:
oops.